api/index.go

```diff
@@ -149,9 +150,15 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 
 //main scraping function
 func scrapeMain(users []string, intersect bool, ignoreList toIgnore) (film, error) {
+
 	var user int = 0          //conuter for number of users increses by one when a users page starts being scraped decreses when user has finished think kinda like a semaphore
 	var totalFilms []film     //final list to hold all film
 	ch := make(chan filmSend) //channel to send films over
+	go func() {
+		time.Sleep(55 * time.Second)
+		log.Println("timeout")
```

Comment:
I think you should log something more explicit (like, "WARN: maximum time reached, sending whatever we got").

Reply:
If you want I can switching logging to https://github.com/uber-go/zap for faster and more structured logging

Reply:
Added better logging let me know if you want other things logged